test/gurka/test_conditional_restrictions.cc

```diff
@@ -0,0 +1,69 @@
+#include "gurka.h"
+#include <gtest/gtest.h>
+
+using namespace valhalla;
+
+class ConditionalRestrictions : public ::testing::Test {
+protected:
+  static gurka::map map;
+
+  static void SetUpTestSuite() {
+    constexpr double gridsize = 100;
+
+    const std::string ascii_map = R"(
+         A--B
+         |  \
+         D--C)";
+
+    const gurka::ways ways = {
+        {"AD", {{"highway", "service"}, {"motorcar", "no"}, {"motor_vehicle", "no"}}},
+        {"AB",
+         {{"highway", "service"},
+          {"motorcar:conditional",
+           "yes @ (Mar 00:00-07:00);Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00;Sep 00:00-07:00;Sep 18:00-24:00;Oct 00:00-07:00;Oct 17:00-24:00;Jan-Feb 00:00-07:30;Jan-Feb 17:00-24:00;May-Jul 00:00-07:00;May-Jul 20:00-24:00)"},
+          {"motor_vehicle:conditional",
+           "yes @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00;Sep 00:00-07:00;Sep 18:00-24:00;Oct 00:00-07:00;Oct 17:00-24:00;Jan-Feb 00:00-07:30;Jan-Feb 17:00-24:00;May-Jul 00:00-07:00;May-Jul 20:00-24:00)"}}},
+        {"BC",
+         {{"highway", "service"},
+          {"motorcar:conditional",
+           "yes @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00;Sep 00:00-07:00;Sep 18:00-24:00;Oct 00:00-07:00;Oct 17:00-24:00;Jan-Feb 00:00-07:30;Jan-Feb 17:00-24:00;May-Jul 00:00-07:00;May-Jul 20:00-24:00)"},
+          {"motor_vehicle:conditional",
```

Comment:
not sure why you have both a motorcar:conditional and motor_vehicle:conditional.  They are the same.  Other than that, not sure why this is failing.   I guess it will have to be debugged.